stdlib/websub/src/main/ballerina/websub/hub_configuration.bal

```diff
@@ -26,9 +26,10 @@ import ballerina/log;
 @final string DEFAULT_SIGNATURE_METHOD = "SHA256";
 
 //TODO: Fix persistence configs, H2?
-@final string DEFAULT_DB_URL = "jdbc:mysql://localhost:3306/subscriptionsdb";
-@final string DEFAULT_DB_USERNAME = "ballerina";
-@final string DEFAULT_DB_PASSWORD = "ballerina";
+@final string DEFAULT_DB_DIRECTORY = "/tmp/weubsubdb";
+@final string DEFAULT_DB_NAME = "SUBSCRIPTIONS_DB";
```

Comment:
Not related to your change, but would it be possible to make this `HUB_DB` instead of `SUBSCRIPTIONS_DB`? :)